docs/en/latest/plugin-develop.md

```diff
@@ -42,6 +42,11 @@ For example, you can create a directory structure like this:
 │           └── plugins
 │               └── 3rd-party.lua
 ```
+:::note
+
+custom plugin root directory must exists `/apisix/plugins` subdirectory**
```

Comment:
Please refer to Chinese to modify the English document.

Reply:
ok